Header header
uint8 enable_gps_test
uint8 enable_rf_test
uint8 full_test
uint8 end_test
